packages/shared/components/popups/AddProposalPopup.svelte

```diff
@@ -64,12 +66,15 @@
     }
 
     async function registerParticipationWrapper(auth?: Auth): Promise<void> {
-        await registerParticipationEvent(eventId, [{ url: nodeUrl, auth }])
+        const event = await registerParticipationEvent(eventId, [{ url: nodeUrl, auth }])
         showAppNotification({
             type: 'success',
             message: localize('views.governance.proposals.successRegister'),
             alert: true,
         })
+        const proposal = await createProposalFromEvent(event)
```

Comment:
Don't need to await here.
```suggestion
        const proposal = createProposalFromEvent(event)
```